.github/workflows/build_doc.yml

```diff
@@ -53,9 +53,7 @@ jobs:
           path: build_doc
 
       - name: Deploy
-        uses: peaceiris/actions-gh-pages@v4
+        uses: JamesIves/github-pages-deploy-action@v4
         with:
-          deploy_key: ${{ secrets.ACTIONS_DEPLOY_KEY }}
-          publish_dir: build_doc
-          destination_dir: dev
-          enable_jekyll: true
+          folder: build_doc
+          target-folder: dev
```

Comment:
We used to have:

```
      # Deploy files contain in _build directory on gh-pages branch
      - name: deploy-doc
        uses: JamesIves/github-pages-deploy-action@3.6.2
        with:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          BRANCH: gh-pages
          FOLDER: _build
          CLEAN: false
```

does that no longer work?

Reply:
Also possibly you need to deploy from a branch in the repository (not from a fork)?

Reply:
You can try directly on your fork if you enable GH actions on your fork (although you may need to enable GH pages as well, not sure)

Reply:
Both does not work so it seems we cannot deploy from a PR I guess. I will try on my fork.

Reply:
It works on my fork... aahhhhhhhhhhhhhhhh